src/inputs/pipe.c

```diff
@@ -302,6 +305,42 @@ parse_progress(struct input_metadata *m, char *progress)
   m->song_length = (end - start) * 10 / 441; // Convert to ms based on 44100
 }
 
+static void
+parse_volume(char *volume)
+{
+  char *value_begin;
+  char *value_end;
+  float airplay_volume;
+
+  if (!(value_begin = strtok_r(volume, ",", &value_end)))
+    return;
+
+  airplay_volume = strtof(value_begin, &value_end);
+
+  if ((errno == ERANGE) || (errno != 0 && airplay_volume == 0) || (value_begin == value_end))
+    {
+      DPRINTF(E_LOG, L_PLAYER, "Invalid airplay volume in string (%s): %s\n", volume, strerror(errno));
+      return;
+    }
+
+  if (fabs(airplay_volume + 144) < 1)
```

Comment:
The spec says "-144" means mute and so I just tried to stick to the speck (including comparison of floating points as range instead of as exact value). "< -30" would also do the trick yes but it might then also mute if some invalid content is transmitted.